go/pkg/client/cas_test.go

```diff
@@ -1867,3 +1867,51 @@ func TestDownloadFilesCancel(t *testing.T) {
 		})
 	}
 }
+
+func TestBatchDownloadBlobsCompressed(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	listener, err := net.Listen("tcp", ":0")
+	if err != nil {
+		t.Fatalf("Cannot listen: %v", err)
+	}
+	fakeCAS := fakes.NewCAS()
+	defer listener.Close()
```

Comment:
`t.Cleanup` is probably more suited than `defer` in tests.

Reply:
https://stackoverflow.com/questions/61609085/what-is-useful-for-t-cleanup

Sounds like t.Cleanup is useful when the test calls other functions (which doesn't happen here). In addition, all other tests here also just defer the cleanup function. I would rather it stay like the other tests and fix them all in a separate change.

Reply:
Fair point.